common/buildings/wc_cultural_tradepost.txt

```diff
@@ -0,0 +1,314 @@
+		}
+		gold_cost = 300
+		build_time = 600
+		ai_creation_factor = 95
+		
+		retinuesize = 30
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Tuskar Group
+	tp_turtle_pools = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = tuskarr_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 300
+		build_time = 600
+		ai_creation_factor = 95
+		
+		retinuesize = 30
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Orc Group
+	tp_wolf_bestiary = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = orc_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 300
+		build_time = 600
+		ai_creation_factor = 95
+		
+		retinuesize = 30
+		military_techpoints = 0.01
+		tradevalue = 10
+		trade_route_wealth = 1		
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Harpy Group
+	tp_guide_eagle_nests = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = harpy_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 300
+		build_time = 625
+		ai_creation_factor = 95
+		
+		retinuesize = 20
+		tech_growth_modifier = 0.2		
+		tradevalue = 15
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+		
+	##Troll Group
+	tp_voodo_charm_vendor = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = harpy_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 350
+		build_time = 700
+		ai_creation_factor = 95
+		
+		cultural_techpoints = 0.02	
+		tradevalue = 20
+		trade_route_wealth = 2	
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Dwarfish Group
+	tp_dwarfish_stonemasons = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = harpy_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 350
+		build_time = 700
+		ai_creation_factor = 95
+
+		garrison_size = 1.5		
+		retinuesize = 10
+		military_techpoints = 0.01	
+		tradevalue = 20
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Kobolt Group
+	tp_chandlers_guild = {
+		potential = { in_major_trade_route_trigger = yes }
+		desc = tp_empty
+		trigger = {
+			FROM = { culture_group = harpy_group }
+			TECH_TRADE_PRACTICES = 3			
+		}
+		gold_cost = 300
+		build_time = 550
+		ai_creation_factor = 95
+
+		monthly_character_piety = 0.01
+		economic_techpoints = 0.01
+		cultural_techpoints = 0.01
+		tradevalue = 20
+		trade_route_wealth = 2			
+		
+		extra_tech_building_start = 10.0 # Never get at start of game
+	}
+	
+	##Goblins Group

```

Comment:
It says Goblins though it's gnome building.

Reply:
solved